pip_audit/_cli.py

```diff
@@ -330,8 +331,14 @@ def _dep_source_from_project_path(
     project_path: Path, resolver: ResolveLibResolver, state: AuditState
 ) -> DependencySource:  # pragma: no cover
     # Check for a `pyproject.toml`
```

Comment:
Looks like this comment needs to be moved down/removed entirely (it's a little redundant).

Reply:
removed ✅